server/datastore/mysql/scripts_test.go

```diff
@@ -517,3 +518,102 @@ VALUES
 		}
 	})
 }
+
+func testBatchSetScripts(t *testing.T, ds *Datastore) {
+	ctx := context.Background()
+
+	applyAndExpect := func(newSet []*fleet.Script, tmID *uint, want []*fleet.Script) map[string]uint {
+		err := ds.BatchSetScripts(ctx, tmID, newSet)
+		require.NoError(t, err)
+
+		if tmID == nil {
+			tmID = ptr.Uint(0)
+		}
+		got, _, err := ds.ListScripts(ctx, tmID, fleet.ListOptions{})
+		require.NoError(t, err)
+
+		// compare only the fields we care about
+		m := make(map[string]uint)
+		for _, gotp := range got {
```

Comment:
Nit: "gotp" -> "gotS" or "s" (I think "p" is because was based on the profiles tests?)